# Housing Market

### Introduction:

This time we will create our own dataset with fictional numbers to describe a house market. As we are going to create random data don't try to reason of the numbers.

### Step 1. Import the necessary libraries

In [1]:
import pandas as pd
import numpy as np

### Step 2. Create 3 differents Series, each of length 100, as follows:
1. The first a random number from 1 to 4
2. The second a random number from 1 to 3
3. The third a random number from 10,000 to 30,000

In [2]:
series1 = pd.Series(np.random.randint(1, 5, size=100), name='Series1')
series2 = pd.Series(np.random.randint(1, 4, size=100), name='Series2')
series3 = pd.Series(np.random.randint(10000, 30001, size=100), name='Series3')

### Step 3. Let's create a DataFrame by joinning the Series by column

In [5]:
df1 = series1.to_frame()
df2 = series2.to_frame()
df3 = series3.to_frame()

df = df1.join(df2).join(df3)

### Step 4. Change the name of the columns to bedrs, bathrs, price_sqr_meter

In [7]:
df.columns = ['bedrs', 'bathrs', 'price_sqr_meter']
df

bedrs  bathrs  price_sqr_meter
0       4       2            10053
1       2       3            27625
2       1       2            29280
3       1       3            17413
4       3       3            28636
..    ...     ...              ...
95      1       2            14028
96      4       3            20367
97      2       2            24811
98      1       3            26452
99      4       3            24089

[100 rows x 3 columns]

### Step 5. Create a one column DataFrame with the values of the 3 Series and assign it to 'bigcolumn'

In [9]:
bigcolumn = pd.concat([series1, series2, series3], axis=0).reset_index(drop=True)

bigcolumn = bigcolumn.to_frame(name='bigcolumn')

bigcolumn

bigcolumn
0            4
1            2
2            1
3            1
4            3
..         ...
295      14028
296      20367
297      24811
298      26452
299      24089

[300 rows x 1 columns]

### Step 6. Oops, it seems it is going only until index 99. Is it true?

In [11]:
# not true

### Step 7. Reindex the DataFrame so it goes from 0 to 299

In [12]:
# it is